umake/frameworks/ide.py

```diff
@@ -477,6 +477,21 @@ def __init__(self, category):
                          icon_filename='product.png')
 
 
+class Rider(BaseJetBrains):
+    """The JetBrains  cross-platform .NET IDE"""
+    download_keyword = 'RD'
+    executable = "rider.sh"
+
+    def __init__(self, category):
+        super().__init__(name="Rider",
+                         description=_("The JetBrains  cross-platform .NET IDE"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         packages_requirements=['openjdk-7-jdk | openjdk-8-jdk'],
```

Comment:
Does this need openjdk?
Being a C# ide I don't think it needs the JDK.
And like other intellij ides it should have a JRE contained in the tarfile